appinfo/app.php

```diff
@@ -77,6 +77,9 @@
 	return;
 }
 
+$app = new \OCA\User_SAML\AppInfo\Application();
+$app->registerDavAuth();
```

Comment:
We have an info.xml entry to register DAV plugins

Reply:
those plugins only get loaded after auth, which is to late here